cfg.lex

```diff
@@ -267,6 +267,7 @@ MCAST_LOOPBACK		"mcast_loopback"
 MCAST_TTL			"mcast_ttl"
 TOS					"tos"
 DISABLE_DNS_FAILOVER  "disable_dns_failover"
+REDACT_SIP_PII "redact_sip_pii"
```

Comment:
We need to eventually redact all PII from logging and if this is ever accepted upstream then other modules may use it too. I think `SIP` may be too specific here, as not all the PII being redacted will be in SIP necessarily (could be from a DB lookup, for example).